frontend/src/lib/components/PropertyFilters/OperatorValueSelect.tsx

```diff
@@ -39,7 +39,7 @@ export function OperatorValueSelect({
                             onChange(newOperator, newOperator)
                         } else if (isOperatorFlag(currentOperator || 'exact')) {
                             onChange(newOperator, null)
-                        } else if (value) {
+                        } else {
```

Comment:
:thinking: I think the mistake is on line 34 instead: it should be `currentOperator` over `operator`.

Keeping this keeps PropertyFilter.js simpler - we don't call onChange until the inputs are 'valid'.

Reply:
yeah let's go with that